scripts/install-multi-user.sh

```diff
@@ -810,6 +810,23 @@ install_from_extracted_nix() {
         _sudo "to make the new store non-writable at $NIX_ROOT/store" \
               chmod -R ugo-w "$NIX_ROOT/store/"
 
+        local get_chr_own="$(PATH="$(getconf PATH 2>/dev/null)" command -vp chown)"
+        if [[ -z "$get_chr_own" ]]; then
+            get_chr_own="$(command -v chown)"
+        fi
+
+        if [[ -z "$get_chr_own" ]]; then
+            reminder <<EOF
+I wanted to change ownership of new Nix store files,
+but I couldn't locate 'chown'. (You may need to fix your PATH.)
+To manually change file ownership, you can run:
+    sudo chown -R 'root:$NIX_BUILD_GROUP_NAME' '$NIX_ROOT'
+EOF
+        else
+            _sudo "to change ownership of Nix store files" \
+                  "$get_chr_own" -R "root:$NIX_BUILD_GROUP_NAME" "$NIX_ROOT" || true
+        fi
```

Comment:
@abathur Do you think it would be worthwhile to factor this out into a function out, since it's also done in the case that the store already exists?

Reply:
I'm of two minds:
1. Broadly/normally: yes
2. A restructuring will probably take more testing. There's an effort afoot to make a binary installer. It isn't a given it'll succeed and be adopted, but I'm personally inclined to make minimum-effort fixes until it's clear how that effort will shake out. I think it makes sense to leave a comment and wait for a third repetition to actually extract it?

Reply:
> A restructuring will probably take more testing.

We could just make a function like `chown_store` and pass a parameter to adjust the message accordingly if we really want, though -- I don't think it would be that much of a stretch.

> I think it makes sense to leave a comment and wait for a third repetition to actually extract it?

Sure, makes sense. How's what I just pushed?